build.sbt

```diff
@@ -1,5 +1,12 @@
 import Dependencies.Compile._
```

Comment:
StatusCodeToStatusCode.scala seems empty

Reply:
Thanks! Removed in https://github.com/47deg/TBD/pull/37/commits/6a61be117d4460edcd578cdbe8f184a5fde740b1